server/test/buff_server_grpc/auth_service_test.exs

```diff
@@ -1,42 +1,50 @@
 defmodule BuffServerGrpc.AuthServiceTest do
+  use BuffServerGrpc.IntegrationCase
   use BuffServer.DataCase
 
   alias BuffServer.Authentication.Token, as: AuthToken
+  alias BuffServerGrpc.AuthService
   alias BuffServerGrpc.AuthService.Server, as: AuthServer
 
   describe "AuthService" do
     setup :setup_user_fixture
 
-    test "should return an error for incorrect username" do
-      assert_raise(GRPC.RPCError, fn ->
-        AuthServer.login(%{username: "a", password: "c"}, nil)
+    test "should return an error for incorrect email" do
+      get_client(AuthServer, fn channel ->
```

Comment:
That's the magic here. this actually uses the client..
It could be rewritten in a cleaner way using macros, but this should be good enough.

Reply:
Well, the current syntax is very verbose, in the future it would be cool to see how it can be minimized, maybe through setup + tags